Packs/AWS-IAM/Integrations/AWS-IAM/AWS_IAM_test.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+import importlib
+import demistomock as demisto
+from CommonServerPython import tableToMarkdown, pascalToSpace
+
+AWS_IAM = importlib.import_module("AWS-IAM")
+
+ATTACHED_POLICIES = [
+    {
+        'PolicyName': 'AdministratorAccess',
+        'PolicyArn': 'arn:aws:iam::aws:policy/AdministratorAccess'
+    }
+]
+
+ATTACHED_RESPONSE = {
+    'AttachedPolicies': ATTACHED_POLICIES,
+    'IsTruncated': True,
+    'Marker': '111'
+}
+
+PAGINATION_CHECK = [
+    (
+        {'userName': 'test'},
+        ['AllAccessPolicy', 'KeyPolicy']),
+    (
+        {
+            'userName': 'test',
+            'page': 2,
+            'page_size': 1,
+            'marker': '111'
+        },
+        ['KeyPolicy'])
+]
+
+
+class AWSClient:
+    def aws_session(self):
+        pass
+
+
+class Boto3Client:
+    def list_user_policies(self):
+        pass
+
+    def list_attached_user_policies(self):
+        pass
+
+    def list_attached_group_policies(self):
+        pass
+
+    def get_login_profile(self):
+        pass
+
+
+@pytest.mark.parametrize('args, res', PAGINATION_CHECK)
+def test_list_user_policies(mocker, args, res):
+    response = {
+        'PolicyNames': [
+            'AllAccessPolicy',
+            'KeyPolicy'
+        ],
+        'IsTruncated': True,
+        'Marker': '111'
+    }
+
+    policy_data = []
+    for policy in res:
+        policy_data.append({
+            'UserName': 'test',
+            'PolicyName': policy,
+        })
+
+    ec = {'AWS.IAM.UserPolicies(val.PolicyName && val.UserName && val.PolicyName === obj.PolicyName && '
+          'val.UserName === obj.UserName)': policy_data,
+          'AWS.IAM.Users(val.UserName === \'{}\').InlinePoliciesMarker'.format('test'): response.get('Marker')}
```

Comment:
you are pretty much providing the same code lines as the function you are testing, no need to do all of this. Just check if the mock data exists in the response.